README.md

```diff
@@ -8,7 +8,7 @@
 [![tests](https://github.com/napari/napari-plugin-manager/workflows/test_and_deploy/badge.svg)](https://github.com/napari/napari-plugin-manager/actions)
 [![codecov](https://codecov.io/gh/napari/napari-plugin-manager/branch/main/graph/badge.svg)](https://codecov.io/gh/napari/napari-plugin-manager)
 
-A plugin that adds a plugin manager to [napari].
+[napari] plugin manager to enable installing [napari] packages, that is itself a plugin.
```

Comment:
```suggestion
[napari] plugin manager to enable installing [napari] plugins, that is itself a plugin.
```
Not sure we use `package` in this context, so maybe best to keep it simple and use plugin?

Reply:
Good point, don't know what I was thinking! Thanks!